lib/galaxy/webapps/galaxy/api/workflows.py

```diff
@@ -65,6 +65,8 @@ def index(self, trans, **kwd):
             encoded_id = trans.security.encode_id(wf.id)
             item['url'] = url_for('workflow', id=encoded_id)
             item['owner'] = wf.user.username
+            item['latest_workflow_steps'] = len( wf.latest_workflow.steps )
+            item['published'] = wf.published
```

Comment:
This line is not necessary, `'published'` field is already filled by the `to_dict` method.